### PR TITLE
docs: add functions doc

### PR DESCRIPTION
**Pull request by MachaelLee**
add functions doc

### CODE REVIEW

File: docs/src/cn/functions/aggregate_functions.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# 聚合函数
```
Comment (jiacai2050) [inline docs/src/cn/functions/aggregate_functions.md:1]: Remove this file

File: docs/src/cn/sql/README.md [hunk 2]
```diff
@@ -15,3 +15,5 @@
   - [数据查询](dml/select.md)
 - [引擎参数](engine_options.md)
 - [常见 SQL](utility.md)
+- [标量函数](functions/scalar_functions.md)
```
Comment (jiacai2050) [inline docs/src/cn/sql/README.md:18]: sql/functions/aggregate_functions.md ?

File: docs/src/cn/functions/aggregate_functions.md (hunk 1 above)
Comment (MachaelLee) [reply docs/src/cn/functions/aggregate_functions.md]: done

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**